src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private int maxValue = 100;
```

Comment:
it's because you didn't put `static final` keywords, which marks constants
```suggestion
    private static final int MAX_VALUE = 100;
```